tiny-event-sourcing-kafka-spring-boot-starter/pom.xml

```diff
@@ -0,0 +1,286 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <packaging>jar</packaging>
+
+    <name>tiny-event-sourcing-kafka-spring-boot-starter</name>
+    <description>Tiny event sourcing pub/sub kafka autoconfiguration</description>
+    <developers>
+        <developer>
+            <name>
+                Jalal Naghiyev
+            </name>
+        </developer>
+        <developer>
+            <name>
+                Andrey Sukhovitsky
+            </name>
+        </developer>
+        <developer>
+            <name>
+                Sevastyan Trutnev
+            </name>
+        </developer>
+    </developers>
+    <licenses>
+        <license>
+            <name>Apache License 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+        </license>
+    </licenses>
+    <groupId>ru.quipy</groupId>
+    <artifactId>tiny-event-sourcing-kafka-spring-boot-starter</artifactId>
+    <version>2.3.0-SNAPSHOT</version>
+
+    <properties>
+        <java.version>11</java.version>
+        <spring.boot.version>2.6.6</spring.boot.version>
+        <kotlin.version>1.6.21</kotlin.version>
+        <kotlin.compiler.incremental>true</kotlin.compiler.incremental>
+        <tiny-event-sourcing-lib.version>2.3.0-SNAPSHOT</tiny-event-sourcing-lib.version>
+        <tiny-event-sourcing-kafka.version>2.3.0-SNAPSHOT</tiny-event-sourcing-kafka.version>
+        <tiny-event-sourcing-spring-boot-starter.version>2.3.0-SNAPSHOT</tiny-event-sourcing-spring-boot-starter.version>
+        <tiny-mongo-storage-spring-boot-starter.version>2.3.0-SNAPSHOT</tiny-mongo-storage-spring-boot-starter.version>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-dependencies</artifactId>
+                <version>${spring.boot.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>ru.quipy</groupId>
+            <artifactId>tiny-event-sourcing-spring-boot-starter</artifactId>
+            <version>${tiny-event-sourcing-spring-boot-starter.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>ru.quipy</groupId>
+            <artifactId>tiny-event-sourcing-kafka</artifactId>
+            <version>${tiny-event-sourcing-kafka.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>ru.quipy</groupId>
+            <artifactId>tiny-mongo-storage-spring-boot-starter</artifactId>
```

Comment:
I can't find this artifact. Please doublecheck it. Maybe this? tiny-mongo-event-store-spring-boot-starter